app/controllers/v0/search_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module V0
+  class SearchController < ApplicationController
+    skip_before_action :authenticate
+
+    def index
+      response = Search::Service.new(params['query']).results
```

Comment:
Do we want to rely on the front-end for sanitation of queries, or should there be a "strong params" type private method here, to avoid directly passing params into a controller action?

Reply:
Good call